25_exceptions_lab/01.py

```diff
@@ -4,4 +4,22 @@
 If value is negative or not a number show 
 a warning and keep reading values
 """
+from math import sqrt
 
+def mySqrt(val):
+    if not (type(val) == int):
+        raise TypeError("Value is not integer")
+    elif (val < 0):
+        raise ValueError("Value must be greater than 0")
+    return sqrt(val)
```

Comment:
Python's sqrt already raises exceptions, so no need to wrap it in your own function. Just catch the exception and handle them correctly